platform/viewer/src/connectedComponents/ConnectedPluginSwitch.js

```diff
@@ -35,11 +35,13 @@ const mapStateToProps = state => {
 const mergeProps = (propsFromState, propsFromDispatch, ownProps) => {
   //const { activeViewportIndex, layout } = propsFromState;
   //const { setLayout } = propsFromDispatch;
+  const { togglePlugSwitchStatus } = ownProps;
 
   // TODO: Do not display certain options if the current display set
   // cannot be displayed using these view types
   const mpr = () => {
-    commandsManager.runCommand('mpr2d');
+        commandsManager.runCommand("mpr2d");
```

Comment:
We could fire a different command if we know the togglePlugSwitchStatus? One that exits that viewing mode? Or, I think we can pass an object of options to the command we're calling "runCommand" for? If not, that shouldn't be a hard change, and you can have a single command that could handle the exit.

Just thinking of ways to reuse the code instead of adding a whole new component to handle this.